src/sentry/issues/endpoints/organization_group_suspect_flags.py

```diff
@@ -0,0 +1,55 @@
+from rest_framework.request import Request
+from rest_framework.response import Response
+
+from sentry import features
+from sentry.api.api_publish_status import ApiPublishStatus
+from sentry.api.base import EnvironmentMixin, region_silo_endpoint
+from sentry.api.bases import GroupEndpoint
+from sentry.api.helpers.environments import get_environments
+from sentry.api.utils import get_date_range_from_params
+from sentry.issues.suspect_flags import get_suspect_flag_scores
+from sentry.models.group import Group
+
+
+@region_silo_endpoint
+class OrganizationGroupSuspectFlagsEndpoint(GroupEndpoint, EnvironmentMixin):
+    publish_status = {"GET": ApiPublishStatus.PRIVATE}
+
+    def get(self, request: Request, group: Group) -> Response:
+        """Stats bucketed by time."""
+        if not features.has(
+            "organizations:feature-flag-suspect-flags",
+            group.organization,
+            actor=request.user,
+        ):
+            return Response(status=404)
+
+        environments = [e.name for e in get_environments(request, group.organization)]
+        group_id = group.id
+        organization_id = group.organization.id
+        project_id = group.project.id
+        start, end = get_date_range_from_params(request.GET)
+
+        # Clamp the range to be within the issue's first and last seen timestamps.
+        start, end = max(start, group.first_seen), min(end, group.last_seen)
+
+        # To increase our cache hit-rate we round the dates down to the nearest 5 minute interval.
```

Comment:
i'm curious if for the baseline query we could cache at the day interval instead of this 5 min interval. something to play with if these queries run into problems

Reply:
That's an interesting idea.  I'd like to keep them aligned for now just because the math is finicky.  Another thought, since the baseline query can be cached from any issue we'll get the benefit globally for that project.  So every 5 minutes someone will have to pay the price but every issue gets the benefit for that cache window.  So 5 minutes might not be so bad.

Reply:
Ah, but I just realized that benefit is going to go away because I might be changing the baseline query to scope it to the selection set more tightly.  We'll see I might have to do the 1 day rounding.